internal/core/src/common/CMakeLists.txt

```diff
@@ -23,7 +23,8 @@ set(COMMON_SRC
         IndexMeta.cpp
         EasyAssert.cpp
         FieldData.cpp
-)
+        Utils.cpp
```

Comment:
rename it to indicate re2 module, utils is to common

Reply:
OK.